pages/index.tsx

```diff
@@ -1,62 +1,105 @@
+        <title> Startify </title>
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
       <main>
-        <h1 className="title">
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
-
-        <p className="description">
-          Get started by editing <code>pages/index.js</code>
-        </p>
-
-        <div className="grid">
-          <a href="https://nextjs.org/docs" className="card">
-            <h3>Documentation &rarr;</h3>
-            <p>Find in-depth information about Next.js features and API.</p>
-          </a>
-
-          <a href="https://nextjs.org/learn" className="card">
-            <h3>Learn &rarr;</h3>
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
-
-          <a
-            href="https://github.com/vercel/next.js/tree/master/examples"
-            className="card"
-          >
-            <h3>Examples &rarr;</h3>
-            <p>Discover and deploy boilerplate example Next.js projects.</p>
-          </a>
-
-          <a
-            href="https://vercel.com/import?filter=next.js&utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-            className="card"
-          >
-            <h3>Deploy &rarr;</h3>
-            <p>
-              Instantly deploy your Next.js site to a public URL with Vercel.
-            </p>
-          </a>
+        <h1 className="title">Welcome to Startify!</h1>
+        <div className="form">
+          <Typography variant="h6">
+            Enter details of your startup to get started!
+          </Typography>
+          <Formik initialValues={initialValues} onSubmit={handleSubmit}>
+            {({ dirty, isValid }) => {
+              return (
+                <Form>
+                  <FormField name="name" label="Company Name" />
+                  <FormSelect
+                    name="type"
+                    items={positionItems}
+                    label="Company Type"
+                  />
+
+                  <FormSelect
+                    name="theme"
+                    items={themeItems}
+                    label="Website Theme?"
+                  />
+
+                  <Button
+                    variant="contained"
+                    color="primary"
+                    disabled={!dirty || !isValid}
+                    type="submit"
+                  >
+                    Generate my website!
+                  </Button>
+                </Form>
+              );
+            }}
+          </Formik>
         </div>
       </main>
 
       <footer>
-        <a
-          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Powered by{' '}
-          <img src="/vercel.svg" alt="Vercel Logo" className="logo" />
-        </a>
+        <p> Made by Team 2 in Pod 0.3.2 </p>
```

Comment:
```suggestion
        <p> Made by Team 1 in Pod 0.3.2 </p>
```
I think we are Team 1